src/Microsoft.Framework.PackageManager/Restore/NuGet/NuGetv2Feed.cs

```diff
@@ -232,17 +240,49 @@ public PackageInfo BuildModel(string id, XElement element)
             });
         }
 
+        private void EnsureValidFindPackagesResponse(XDocument doc)
+        {
+            if (!string.Equals(doc.Root.GetDefaultNamespace().ToString(), _defaultNamespace,
```

Comment:
should be `doc.Root.Name.Namespace`. Currently I believe it would return `abc` for the following xml <ns1:element xmlns="abc" xmlns:ns1="xyz" /> even though `element` is in the `xyz` namespace.